docs/content/quickstart.md

```diff
@@ -108,7 +108,7 @@ instance to the name of the bundle. This time we will explicitly name the
 installation "demo".
 
 ```
-porter install demo --tag deislabs/porter-hello-bundle:latest
+porter install demo --tag getporter/porter-hello-bundle:latest
```

Comment:
We need to be really careful updating the tags under docs/content/slides (we use this for workshops and talks). `getporter/porter-hello-bundle` isn't a published bundle and we don't plan on publishing under that name pattern *-bundle going forward. It should be `getporter/hello:v0.1.0` but that also means we need to fix the tag (latest isn't published v0.1.0 is) and remove the invocation image above.